debug:
  distance_thres: True
  downsample: True
  pppc: True
  horizon: True
  horizon_centroids: True 
  table_plane: True
  table_object_cluser: True
  foreground_centroid: True
downsampling:
  size: 0.005
dist_thres:
  head_dist:
    x_min: 0.0
    x_max: 1.5
    y_min: -1.0
    y_max: 1.0
    z_min: -1.2
    z_max: 0.5
  arm_dist:
  depth: 1.2
filter_inlier:
  axis: "x"
clustering:
  three:
    tolerance: 0.02 # [m]
    min_size: 10
    max_size: 2500
  inlier:
    tolerance: 0.02 # [m]
    min_size: 10
    max_size: 2500
 
